proto/topo.proto

```diff
@@ -81,6 +81,34 @@ message Node {
   // If interfaces is empty the interfaces defined in the links portion of the
   // topology will be populated into the node.
   map<string, Interface> interfaces = 12;
+  // Any host constraints required by the node. These may not be required by a 
+  // vendor, but it can be used to provide requirements for the node.
+  repeated HostConstraint host_constraints = 13;
+}
+
+// HostConstraint is a constraint on the host where the topology is running.
```

Comment:
... on the host where the node is running.

Reply:
Fixed.